model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -635,8 +642,12 @@ def model_output_to_completion_output(
                     tokens=tokens,
                 )
             except Exception as e:
-                logger.exception(f"Error parsing text-generation-inference output {model_output}")
-                raise e
+                logger.exception(f"Error parsing text-generation-inference output {model_output}.") 
+                if model_output.get("error_type") == "validation":
+                    raise InvalidRequestException(model_output.get("error")) # trigger a 400
+                else:
+                    raise UpstreamServiceError(model_output.get("error")) # also change llms_v1.py that will return a 500 HTTPException so user can retry
```

Comment:
can you double check 500s with proper error messages are returned in this code path?

Reply:
I'm pretty sure that my most recent commit didn't get pushed, l don't see any of my changes to `llms_v1.py` showing up in the github UI. I'm really sorry for re-requesting a review pre-emptively. I'll fix these and then reach back out, sorry @yunfeng-scale